functions/src/services/GoogleSheet.ts

```diff
@@ -0,0 +1,358 @@
+const gapi = require('googleapis').google;
+
+export enum GoogleScopes {
+  SpreadSheets = 'https://www.googleapis.com/auth/spreadsheets',
+}
+
+const letters: string[] = [
```

Comment:
This may not work in all situations. Large spreadsheets have column numbering `AA`, `AB` etc. I think you should write a function that constructs the `A1` notation of the column by its sequential number.

Reply:
Will look into this and see if there's a better way.